test/tests_pytorch/accelerators/test_accelerator_connector.py

```diff
@@ -45,7 +45,7 @@
     SingleDeviceStrategy,
 )
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
-from tests.helpers.runif import RunIf
+from tests_pytorch.helpers.runif import RunIf
```

Comment:
If the root dir is `test` then it should match with `test_pytorch`

Reply:
I would say if the sub-folder has a significant name overlap with the parent folder name, there is no need to include it in the sub-folder name...

Reply:
Discussed offline. We are going with both "tests"